packages/ember-glimmer/lib/utils/iterable.ts

```diff
@@ -279,6 +278,8 @@ class ArrayIterable {
       let array: any[] = [];
       iterable.forEach((item: any) => array.push(item));
       return ArrayIterator.from(array, keyFor);
+    } else if (typeof iterable[Symbol.iterator] === 'function') {
+      return ArrayIterator.from(Array.from(iterable), keyFor);
```

Comment:
Don't think `Array.from` is available on IE11

Reply:
I fixed it locally

Reply:
It is true, but every browser that supports symbols also supports `Array.from`, so a single check it enough.